### PR TITLE
Improving error messaging

### PR DESCRIPTION
**Pull request by sullof**
This detects a `sinceTimestamp = null` passed to `getEventResult` and throws an error.
Without this check, the error will be thrown by TronGrid, confusing the developer. 
It also adds check for `size` and `page` parameters.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -177,19 +177,28 @@ export default class TronWeb extends EventEmitter {
...
-        
+
+        if (!utils.isInteger(sinceTimestamp))
+            return callback('Invalid sinceTimestamp provided');
+
+        if (!utils.isInteger(size))
+            return callback('Invalid size provided');
+
+        if(size > 200) {
+            console.info('Defaulting to maximum accepted size: 200');
```
Comment (Kondax) [inline src/index.js]: I'd recommend changing this to a `warn` statement

Comment (sullof) [reply src/index.js]: Ok.

**Comment by sullof** [comment]
I used in this PR also the changes proposed by @serderovsh in https://github.com/tronprotocol/tron-web/pull/137 (since that PR contained a few errors).

**Comment by serderovsh** [comment]
@sullof I did not test, thanks for fixing!

**Pull request merged**